package.json

```diff
@@ -133,7 +133,7 @@
     "build-images": "mkdir -p docs/img/dist && node docs/bin/build-image-config.js && node docs/bin/appropriate-images.js --all",
     "create-image": "node docs/bin/create-image",
     "build": "run-s build-docs && batfish build # invoked by publisher when publishing docs on the publisher-production branch",
```

Comment:
Is the increased size needed for batfish `build` command here as well? This run target may not be needed by most developers, but would be good to update it for completeness

Reply:
Actually that command works without any changes, so no need to change it.